README.md

```diff
@@ -2,6 +2,8 @@
 
 ![centos build](https://www.bcgsc.ca/bamboo/plugins/servlet/wittified/build-status/KNOW-KNOW) ![build](https://github.com/bcgsc/pori_graphkb_api/workflows/build/badge.svg?branch=master) [![codecov](https://codecov.io/gh/bcgsc/pori_graphkb_api/branch/master/graph/badge.svg?token=XDTDIQ5793)](https://codecov.io/gh/bcgsc/pori_graphkb_api) ![node versions](https://img.shields.io/badge/node-10%20%7C%2012%20%7C%2014-blue) [![orientdb versions](https://img.shields.io/badge/orientdb-3.0%20%7C%203.1-blue)](https://www.orientdb.org/)
 
+This repository is part of the [platform for oncogenomic reporting and interpretation](https://github.com/bcgsc/pori).
+
```

Comment:
Is this going to say 'part of the PORI' rather than 'PORI'? Is that how it's meant to be said?